CHANGELOG.md

```diff
@@ -21,6 +21,47 @@ New Features
   });
   ```
 
+- Exposed SDK logger using [loglevel](https://www.npmjs.com/package/loglevel) module to allow for updating logging configuration after calling `Video.connect`, `Video.createLocalAudioTrack`, `Video.createLocalTracks` and `Video.createLocalVideoTrack`. With this feature, logs can now be intercepted at runtime to allow realtime processing of the logs which include but not limited to inspecting the log data and sending it to your own server.
```

Comment:
we should mention that `eventListener` parameter to connect is now deprecated with the implementation of this custom logger, and specify an example that shows usage of logger for listening to events.

Reply:
I think deprecating `eventListener` is out of scope and should be on a separate PR.

Reply:
I think it's okay to deprecate the eventListener in separate PR - I was pushing for doing it in this PR to just be sure that it can be done easily with logLevel.

Reply:
Thanks @makarandp0 . It should be super easy. Please check the integration tests. I added a test that listens for room name. The user can do a similar way when we deprecate eventListener.